jax_tqdm/pbar.py

```diff
@@ -29,7 +32,7 @@ def scan_tqdm(
         Progress bar wrapping function.
     """
 
-    _update_progress_bar, close_tqdm = build_tqdm(n, print_rate, **kwargs)
+    _update_progress_bar, close_tqdm = build_tqdm(n, print_rate, tqdm_type, **kwargs)
```

Comment:
nit: Worth making these keyword arguments, as they've been updated in `build_tqdm`?

Reply:
Oops, I reverted to positional arguments!